docs/3-understanding-dataLoader.md

```diff
@@ -41,7 +41,7 @@ The GraphQL execution engine will always try to execute fields in parallel in or
 
     Using `DBContext` pooling allows us to issue a `DBContext` instance for each field needing one. But instead of creating a `DBContext` instance for every field and throwing it away after using it, we are renting so fields and requests can reuse it.
 
-1. Head over to the `Startup.cs` and replace `services.AddDbContext` with `services.AddPooledDbContextFactory`.
+1. Head over to the `Startup.cs` and add `services.AddPooledDbContextFactory` after `services.AddDbContext`.
```

Comment:
With version 13 and newer I actually would use AddDbContext now as we by default now scope a DB context per field